Excite Truck/UniversalDIT_Defaultstyle.json

```diff
@@ -1432,122 +1407,6 @@
-      "`Axis 0-`": "../#DefaultDevices/DualShock4/Defaultstyle/Left X-.png",
-      "`Axis 0+`": "../#DefaultDevices/DualShock4/Defaultstyle/Left X+.png",
-      "`Axis 1-`": "../#DefaultDevices/DualShock4/Defaultstyle/Left Y+.png",
-      "`Axis 1+`": "../#DefaultDevices/DualShock4/Defaultstyle/Left Y-.png",
-      "`Axis 2-`": "../#DefaultDevices/DualShock4/Defaultstyle/Right X-.png",
-      "`Axis 2+`": "../#DefaultDevices/DualShock4/Defaultstyle/Right X+.png",
-      "`Axis 3+`": "../#DefaultDevices/DualShock4/Defaultstyle/Right Y-.png",
-      "`Axis 3-`": "../#DefaultDevices/DualShock4/Defaultstyle/Right Y+.png",
-      "`Full Axis 4+`": "../#DefaultDevices/DualShock4/Defaultstyle/L2.png",
-      "`Full Axis 5+`": "../#DefaultDevices/DualShock4/Defaultstyle/R2.png"
-    },
-    "SDL/0/DualSense Wireless Controller": {
-      "SDL/0/DualSense Wireless Controller": "../#DefaultDevices/DualShock4/Device/Icon.png",
-      "`Button 0`": "../#DefaultDevices/DualShock4/Defaultstyle/Cross.png",
-      "`Button 1`": "../#DefaultDevices/DualShock4/Defaultstyle/Circle.png",
-      "`Button 2`": "../#DefaultDevices/DualShock4/Defaultstyle/Square.png",
-      "`Button 3`": "../#DefaultDevices/DualShock4/Defaultstyle/Triangle.png",
-      "`Button 4`": "../#DefaultDevices/DualShock4/Defaultstyle/Share.png",
-      "`Button 5`": "../#DefaultDevices/DualShock4/Defaultstyle/PS.png",
-      "`Button 6`": "../#DefaultDevices/DualShock4/Defaultstyle/Options.png",
-      "`Button 7`": "../#DefaultDevices/DualShock4/Defaultstyle/L3.png",
-      "`Button 8`": "../#DefaultDevices/DualShock4/Defaultstyle/R3.png",
-      "`Button 9`": "../#DefaultDevices/DualShock4/Defaultstyle/L1.png",
-      "`Button 10`": "../#DefaultDevices/DualShock4/Defaultstyle/R1.png",
-      "`Button 11`": "../#DefaultDevices/DualShock4/Defaultstyle/Pad N.png",
-      "`Button 12`": "../#DefaultDevices/DualShock4/Defaultstyle/Pad S.png",
-      "`Button 13`": "../#DefaultDevices/DualShock4/Defaultstyle/Pad W.png",
-      "`Button 14`": "../#DefaultDevices/DualShock4/Defaultstyle/Pad E.png",
-      "`Button 15`": "../#DefaultDevices/DualShock4/Defaultstyle/Touchpad.png",
-      "`Axis 0-`": "../#DefaultDevices/DualShock4/Defaultstyle/Left X-.png",
-      "`Axis 0+`": "../#DefaultDevices/DualShock4/Defaultstyle/Left X+.png",
-      "`Axis 1-`": "../#DefaultDevices/DualShock4/Defaultstyle/Left Y+.png",
-      "`Axis 1+`": "../#DefaultDevices/DualShock4/Defaultstyle/Left Y-.png",
-      "`Axis 2-`": "../#DefaultDevices/DualShock4/Defaultstyle/Right X-.png",
-      "`Axis 2+`": "../#DefaultDevices/DualShock4/Defaultstyle/Right X+.png",
-      "`Axis 3+`": "../#DefaultDevices/DualShock4/Defaultstyle/Right Y-.png",
-      "`Axis 3-`": "../#DefaultDevices/DualShock4/Defaultstyle/Right Y+.png",
-      "`Full Axis 4+`": "../#DefaultDevices/DualShock4/Defaultstyle/L2.png",
-      "`Full Axis 5+`": "../#DefaultDevices/DualShock4/Defaultstyle/R2.png"
```

Comment:
The changes to the devices must be reversed, these are not up to date.

Reply:
Hi there, I'm not sure how this drifted out-of-date, I just took the latest UniversalDIT_Defaultstyle.json file from the #Assets folder and used it as a base. Does this appear the same there?

Reply:
@Venomalia I will approve the changes to push this through, but all of these devices that you've marked for deletion appear to exist under the UniversalDIT_Defaultstyle.json.

Reply:
I have not marked them for deletion, this means that your changes will delete them when they are merged into the main package.